python3eval/pythoneval.cs

```diff
@@ -333,6 +346,26 @@ private static string PythonStandardLibPath()
             }
             return pythonLibDir;
         }
+
+        public IronPython3Evaluator(IConfiguration config = null)
```

Comment:
Maybe it does not matter in a package, but is it necessary to be public ?

Reply:
I think it could be internal yes - though I don't make any promises about package apis in anyway.